### DISCUSSION TITLE
병원목록 데이터 없음 문제 해결 후

### DISCUSSION THREAD

**Discussion by presso22** [changdoc/naver-vaccine-macro#269] [Q&A]
Id값이랑 orgcode 넣어서 평소랑 똑같이 해봤는데
![aaaaa22](https://user-images.githubusercontent.com/88812138/129178352-8b841bbc-3421-4521-9083-bd0cb64bfa3a.png)

이렇게 나옵니다.
정상적이라면 "자동예약시작" 버튼 누른후 
"자동 예약 정지(ESC)" 버튼이 나와야하는데

그냥 "예약완료"이 상태로 멈춰있습니다.
이건 어떻게 해결해야하나요?

어제는 정상적으로 잘됐는데
오늘 목록없음 뜨고 해결이 안되니 답답하네요.

**Comment by changdoc** [comment]
사용하고 계신 버전이 크롬스토어에서 받은 버전이라면
https://github.com/changdoc/naver-vaccine-macro/releases/tag/v1.1.6
크롬에서 설치하신 프로그램은 삭제하시고 1.1.6버전으로 설치하시고 진행해봐주세요.
최신 버전 설치 방법은 https://github.com/changdoc/naver-vaccine-macro/issues/29 <--- 여기 있습니다.

**Reply by presso22** [reply]
기존버전 삭제 후 최신버전 다운받았고, #29 보고 설치방법 확인중인데 여기서는 zip파일 다운로드 하는방법만 나와있네요ㅠ 개발자 모드 관련 블로그 보아도 저는 개발자모드에 + 표시가 없어서 이게 맞는지 확인이 안됩니다. 어떻게 설치해야하나요?
![개발자](https://user-images.githubusercontent.com/88812138/129181104-d424b4a7-0345-467f-86ee-901539d76ec7.png)
